Tests/Svg.UnitTests/SvgTestHelper.cs

```diff
@@ -231,8 +264,9 @@ protected virtual void CheckSvg(SvgDocument svgDoc, Image img)
             using (var ms = new MemoryStream())
             {
                 img.Save(ms, ImageFormat.Png);
+				img.Save($"d:\\temp\\{Guid.NewGuid().ToString()}.png");
```

Comment:
This line was probably for debugging - it leads to a few failing tests because of missing drive.
If removing it, all tests, but the one that should fail pass.

Reply:
Whoops, good catch. I did that to see the output results of some of the tests and did not spot that debug code. Removed it from the code.